components/Stylesheet/OpenProps.astro

```diff
@@ -0,0 +1,93 @@
+---
+import {OpenProps} from './types'
+
+const {props} = Astro.props as OpenProps
+
+const openPropsURLS = {
```

Comment:
looks like Openprops support is on it's way 🎉

Reply:
You caught me lol